src/index.ts

```diff
@@ -386,9 +386,79 @@ export async function install(
     chunkFileNames: 'common/[name]-[hash].js',
   };
   if (Object.keys(depObject).length > 0) {
-    const packageBundle = await rollup.rollup(inputOptions);
-    await packageBundle.write(outputOptions);
+    // Main Snowpack Install 🚀
+    try {
+      const packageBundle = await rollup.rollup(inputOptions);
+      await packageBundle.write(outputOptions);
+    } catch (err) {
+      // Handle Snowpack Install error
+      const invalidFile = err.loc.file as string;
+
+      // handle invalid files
+      if (invalidFile) {
+        const ext = invalidFile.substr(invalidFile.lastIndexOf('.')).toLowerCase();
+        const relativeName = invalidFile.replace(cwd, '');
+        const suggest = (suggestion: string) =>
```

Comment:
can you move this into it's own function outside of the install() method? Also, it would be worth the effort now to store these `extension -> error-message` mappings in an object or Map so that we (and external contributors) can easily make changes in the future.